.github/workflows/test-application.yaml

```diff
@@ -1,6 +1,10 @@
 name: Test application
 
-on: pull_request
+on:
+    pull_request:
+    push:
+        branches:
+            - master
```

Comment:
Should we also run `release/2.0` here /cc @danrot

Reply:
I mainly did this to be able to display the current status of the master branch in the readme. but it might make sense to do this for release branches too, yes. dont have a opinion here 🙂

Reply:
If we do it, I would also do it for both branches. Maybe even use `release/**` to do that for all release branches. But maybe check on your forked repository first how that behaves, i.e. if this is also executed for branches that do not have a github actions configuration.

Reply:
I added `release/**` to the configuration. But I am not sure how I am able to test the behaviour? 🙈

Reply:
Well, I thought you have activated github actions as well in your repo... But we can also try to merge and see what happens.